### DISCUSSION TITLE
Control bosch intelligent filling loop?

### DISCUSSION THREAD

**Discussion by ricostuart** [emsesp/EMS-ESP32#1756] [Q&A]
i have a greenstar 8000 installed with the added bosch intelligent filing loop which helps remotely fill up the system if there is a detected low pressure occurance (with confirmation before filling happens). Is there a way of being informed via the gateway ems v2 and homeassistant? Would like to get a warning come up if low pressure is detected and that I can remotely increase pressure if required (ie if i am away but the wife is at home to check for leaks ,etc)

**Comment by proddy** [comment]
You could create a trigger in HA that checks the value of `sensor.boiler_syspress`?

**Comment by ricostuart** [comment]
I do have a trigger set in HA as a warning message, but I can’t find a way of controlling the filling loop however.

**Reply by proddy** [reply]
You could remotely force an initiation of the filling process (I'm guessing Worcester has an app or something) and then use EMS-ESP to monitor all the commands on the EMS bus. If there are, we can simulate that action (https://emsesp.github.io/docs/Troubleshooting/#im-missing-certain-data-from-an-ems-device) but I'm doubtful as it's probably all tightly integrated inside the boiler.
